source/directives/select-field.ts

```diff
@@ -298,6 +302,7 @@ module formFor {
 
           // Rather than sanitizing `$scope.model.bindable` to undefined, update the empty option's value.
           // This way users are able to choose between undefined, null, and empty string ~ see #141
+          $scope.model.bindable = formForConfiguration_.defaultSelectEmptyOptionValue;
```

Comment:
No problem on the delay. Work has been super busy for me so I've been distracted anyway. We'll keep our eyes out for anything that might come from this, but for now...it's out there :)